src/app/nav-header/nav-header.component.html

```diff
@@ -1,7 +1,7 @@
 <nav class="navbar navbar-expand-md navbar-dark fixed-top" *ngIf="navService?.navbarVisible$ | async">
     <div class="container-fluid">
       <a class="sr-only sr-only-focusable focus-visible" href="javascript:void(0);" (click)="moveFocus()">Skip to main content</a>
-      <a class="navbar-brand" routerLink="/library" routerLinkActive="active"><i class="fa fa-home" aria-hidden="true"></i><span class="phone-hidden"> Kavita</span></a>
+      <a class="navbar-brand" routerLink="/library" routerLinkActive="active"><img class="logo" src="../../assets/images/logo.png"/><span class="phone-hidden"> Kavita</span></a>
```

Comment:
add an alt="kavita icon" and put aria-hidden="true" on the image itself so it doesn't needlessly announce to screen readers.

Reply:
Added in [latest commit](https://github.com/Kareadita/Kavita-webui/pull/260/commits/088fb0dc3aa07c10f9eb9e3260bbc7b099ca2e3d).

Marking as resolved.